test-kitchen.gemspec

```diff
@@ -59,5 +59,7 @@ Gem::Specification.new do |gem|
   # style and complexity libraries are tightly version pinned as newer releases
   # may introduce new and undesireable style choices which would be immediately
   # enforced in CI
-  gem.add_development_dependency "finstyle",  "1.5.0"
+  #
+  # Replacing finstyle in favor of chefstyle
+  gem.add_development_dependency "chefstyle", "0.4.0"
```

Comment:
Do we need to peg a specific version here? We float on latest in our other Ruby projects like Chef, ChefDK, Omnibus etc.